client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -67,42 +90,130 @@ const CancelPurchaseRefundInformation = ( { purchase, includedDomainPurchase } )
 
 				showSupportLink = false;
 			} else if ( includedDomainPurchase && isDomainRegistration( includedDomainPurchase ) ) {
-				text = [
-					i18n.translate(
-						'This plan includes the custom domain, %(domain)s, normally a %(domainCost)s purchase. ' +
-							'The domain will not be removed along with the plan, to avoid any interruptions for your visitors. ',
-						{
-							args: {
-								domain: includedDomainPurchase.meta,
-								domainCost: includedDomainPurchase.priceText,
-							},
-						}
-					),
-					i18n.translate(
-						'You will receive a partial refund of %(refundAmount)s which is %(planCost)s for the plan ' +
-							'minus %(domainCost)s for the domain.',
-						{
-							args: {
-								domainCost: includedDomainPurchase.priceText,
-								planCost: purchase.priceText,
-								refundAmount: purchase.refundText,
-							},
-						}
-					),
-				];
-
 				if ( isRefundable( includedDomainPurchase ) ) {
-					text.push(
+					text = [
+						i18n.translate(
+							'This plan includes the custom domain, %(domain)s, normally a %(domainCost)s purchase. ' +
+								'Cancelling the domain could cause interruptions for your visitors.',
+							{
+								args: {
+									domain: includedDomainPurchase.meta,
+									domainCost: includedDomainPurchase.priceText,
+								},
+							}
+						),
+						<FormLabel key="keep_bundled_domain">
+							<FormRadio
+								name="keep_bundled_domain_false"
+								value="keep"
+								checked={ ! cancelBundledDomain }
+								onChange={ onCancelBundledDomainChange }
+							/>
+							<span>
+								{ i18n.translate( 'Just cancel the plan and keep %(domain)s.', {
+									args: {
+										domain: includedDomainPurchase.meta,
+									},
+								} ) }
+								<br />
+								{ i18n.translate(
+									'You will receive a partial refund of %(refundAmount)s which is %(planCost)s for the plan ' +
+										'minus %(domainCost)s for the domain. The domain will remain registered to you and can be ' +
+										'used on WordPress.com or transferred.',
+									{
+										args: {
+											domainCost: includedDomainPurchase.priceText,
+											planCost: purchase.priceText,
+											refundAmount: purchase.refundText,
+										},
+									}
+								) }
+							</span>
+						</FormLabel>,
+						<FormLabel key="cancel_bundled_domain">
+							<FormRadio
+								name="cancel_bundled_domain_false"
+								value="cancel"
+								checked={ cancelBundledDomain }
+								onChange={ onCancelBundledDomainChange }
+							/>
+							<span>
+								{ i18n.translate( 'Cancel the plan and the domain "%(domain)s."', {
+									args: {
+										domain: includedDomainPurchase.meta,
+									},
+								} ) }
+								<br />
+								{ i18n.translate(
+									"You will be refunded %(planCost)s, but the domain will be canceled and can't be " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'You will be refunded <strong>%1$s</strong>, but the domain will be canceled and can't be guaranteed to be available for registration again.' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/21628-allow-bundled-domain-refund)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).